src/components/EditCustomer.js

```diff
@@ -1,31 +1,48 @@
 import React, { Component } from "react";
 import { Modal, Button, Row, Col, Form } from "react-bootstrap";
+import "weather-icons/css/weather-icons.css";
+import "bootstrap/dist/css/bootstrap.min.css";
 import Snackbar from "@material-ui/core/SnackBar";
 import IconButton from "@material-ui/core/IconButton";
+
 export class EditCustomer extends Component {
   constructor(props) {
     super(props);
-    this.state = { snackbarOpen: false, snackbarmsg: "" };
+    this.state = {
+      customer: [],
```

Comment:
is this a state for a list of customers (since it is an array) or for a single customer (the name is singular)?